reactive/k8s.py

```diff
@@ -182,6 +183,7 @@ def start_kubelet(etcd):
     status_set('maintenance', 'Starting the Kubernetes services.')
     if is_leader():
         compose.up('master')
+        compose.up('proxy')
```

Comment:
I think you need to remove this in the config_changed() method on the master as well.